Classes/Engine/HandlebarsEngine.php

```diff
@@ -167,7 +167,18 @@ protected function getOptions() : array
                 },
                 'lookup' => function ($labels, $key) {
                     return isset($labels[$key]) ? $labels[$key] : '';
-                }
+                },
+                'ifEqual' => function () {
+                    $arguments = func_get_args();
```

Comment:
@dhuf: Have you considered writing the arguments to the method header?

Reply:
Can you write dynamic arguments to the header ? I mean it could be comparing two or more values. `ifEqual a b c d e`